pages/common/identify.md

```diff
@@ -3,6 +3,14 @@
 > Command-line utility of Image Magick project to describe the format and characteristics of one or more image files.
 > More information: <https://imagemagick.org/script/identify.php>.
 
+- Describe the format and basic characteristics of an image:
+
+`identify {{image}}`
```

Comment:
This is our usual syntax. If you want to you can also change the last example to `{{path/to/filelist.csv}}`.
```suggestion
`identify {{path/to/image}}`
```